e2e-tests/development-runtime/cypress/plugins/block-resources.js

```diff
@@ -51,10 +53,28 @@ const blockAssetsForPage = ({ pagePath, filter }) => {
   return null
 }
 
+function blockPageComponent({ path: pageComponentPath }) {
+  const hiddenPath = path.join(
+    path.dirname(pageComponentPath),
+    `_` + path.basename(pageComponentPath)
+  )
+
+  moveAsset(path.join(srcDir, pageComponentPath), path.join(srcDir, hiddenPath))
+  return null
+}
+
 const restore = () => {
-  const globPattern = path.join(publicDir, `/page-data/**`, `_page-data.json`)
-  const hiddenPageDatas = glob.sync(globPattern)
-  hiddenPageDatas.forEach(restorePageData)
+  const hiddenPageDataGlobPattern = path.join(
+    publicDir,
+    `/page-data/**`,
+    `_page-data.json`
+  )
+  const hiddenPageDatas = glob.sync(hiddenPageDataGlobPattern)
+  hiddenPageDatas.forEach(restoreAsset)
+
+  const hiddenPageComponentGlobPattern = path.join(srcDir, `**`, `_*`)
```

Comment:
This can potentially cause problems if someone adds a component starting with an underscore in development-runtime? Highly unlikely though.

Reply:
Didn't think about this, but this is true. Do you think it's worth changing that implementation now or we should be fine (for now)?

Reply:
I think we are good. But should keep this in mind as this will be pretty hard to debug if happens 😅